orbstation/code/antagonists/rulesets_latejoin.dm

```diff
@@ -11,11 +7,22 @@
 			return FALSE
 	return ..()
 
-//////////////////////////////////////////////
-//                                          //
-//           HERETIC SMUGGLER               //
-//                                          //
-//////////////////////////////////////////////
+/// Latejoin Heretic Ruleset
 
 /datum/dynamic_ruleset/latejoin/heretic_smuggler
 	required_enemies = list(1,1,1,1,1,1,1,1,1,1) // the game is supposed to make one of your sac targets a security member
+
+/// Latejoin Contract Killer Ruleset
+/datum/dynamic_ruleset/latejoin/contract_killer
+	name = "Contract Killer"
+	antag_datum = /datum/antagonist/contract_killer
+	antag_flag = ROLE_KILLER_LATEJOIN
+	antag_flag_override = ROLE_CONTRACT_KILLER
+	restricted_roles = list(
+		JOB_AI,
+		JOB_CYBORG,
```

Comment:
Honestly it'd be a little funny if Cyborgs could get this but... maybe for the best if they cannot